README.md

```diff
@@ -23,51 +23,59 @@ the latest version or at least the minimum version mentioned below.
 ### Steps to setup project:
 
 ```bash
-$ git clone git@github.com:axelerant-trainings/drupalcon24-training-solutions.git
+git clone git@github.com:axelerant-trainings/drupalcon24-training-solutions.git
 ```
 Change to the directory of repository.
 
 ```bash
-$ cd drupalcon24-training-solutions
+cd drupalcon24-training-solutions
 ```
+Checkout to `develop` branch.
 
+```bash
+git checkout -b develop origin/develop
```

Comment:
Should it **git checkout develop**?

Reply:
Some how I was unable to do `git checkout develop` and do `git pull origin develop` it was having some conflicts. I deleted the branch and ran above command to fix it. 

May be this `git checkout -b develop origin/develop` can be done as a one time thing (like, git clone we do), then `git checkout develop` will work.

Reply:
I have added some local development steps as a new commit. 
Let's merge this PR and please update accordingly you see things working with your upcoming PRs.